src/components/skills-quiz/SelectJobCard.jsx

```diff
@@ -39,30 +38,43 @@ const SelectJobCard = () => {
               key={job.name}
               role="group"
               aria-label={job.name}
+              className="ml-2 mt-2"
             >
-              <Card className={`${selectedJob === job.name ? 'border border-dark' : null} mt-2 ml-2`}>
-                <Card.Body>
-                  <Card.Title as="h4" className="card-title mb-2">
+              <Card className={`${selectedJob === job.name ? 'border border-dark' : null} h-100`}>
+                <Card.Body className="row">
+                  <div className="col-11">
+                    <Card.Title as="h4" className="card-title mb-2">
+                      <span>
+                        {job.name}
+                      </span>
+                    </Card.Title>
                     <>
-                      {job.name.length > jobsCharactersCutOffLimit
-                        ? `${job.name.substring(0, jobsCharactersCutOffLimit)}...` : job.name}
+                      {!hideLaborMarketData && (job.job_postings?.length > 0
+                        ? (
+                          <div className="text-gray-700">
+                            <p className="m-0 medium-font">
+                              <span style={{ fontWeight: 700 }}>Median U.S. Salary: </span>
+                              ${formatStringAsNumber(job.job_postings[0].median_salary)}
+                            </p>
+                            <p className="m-0 medium-font">
+                              <span style={{ fontWeight: 700 }}>Job Postings: </span>
+                              {formatStringAsNumber(job.job_postings[0].unique_postings)}
+                            </p>
+                          </div>
+                        ) : (
+                          <div className="text-gray-700">
+                            <p className="m-0 medium-font">
+                              <span style={{ fontWeight: 700 }}>Median U.S. Salary: N/A</span>
+                            </p>
+                            <p className="m-0 medium-font">
+                              <span style={{ fontWeight: 700 }}>Job Postings:  N/A</span>
```

Comment:
Can we define these two string `Median U.S. Salary: N/A` and `Job Postings:  N/A` and use those instead in search and select job card components? In test case too

Reply:
Changing the way of condition so have to keep it this way.